### PR TITLE
projectile is configurable to behave either as guided missile or clas…

### PR DESCRIPTION
**Pull request by sdudasko**
…sic bullet

### CODE REVIEW

File: Assets/Scripts/Combat/Projectile.cs [hunk 1]
```diff
@@ -9,12 +9,32 @@ public class Projectile : MonoBehaviour
...
     // Update is called once per frame
     void Update()
     {
         if (target == null) return;
 
-        transform.LookAt(GetAimLocation());
+        if (!projectileFired && !guidedMissile) {
+            savedLocation = GetAimLocation();
+            projectileFired = true;
+
```
Comment (RichardDominik) [inline Assets/Scripts/Combat/Projectile.cs:24]: extra line

File: Assets/Scripts/Combat/Projectile.cs [hunk 2]
```diff
@@ -9,12 +9,32 @@ public class Projectile : MonoBehaviour
...
+        } else if (guidedMissile) {
+            savedLocation = GetAimLocation();
+        }
+        
+        transform.LookAt(savedLocation);
+
+        print("Vector3.Distance(transform.position, savedLocation):" + Vector3.Distance(transform.position, savedLocation));
+
+        if (Vector3.Distance(transform.position, savedLocation) <= 1)
+        {
```
Comment (RichardDominik) [inline Assets/Scripts/Combat/Projectile.cs:26]: code style

**Review by RichardDominik — APPROVED** [review]

File: Assets/Scripts/Combat/Projectile.cs (hunk 2 above)
Comment (sdudasko) [reply Assets/Scripts/Combat/Projectile.cs:26]: I use Visual Studio for that project and it corrects me to do this style. I was thinking of changing it in preferences but, the change is life:)

File: Assets/Scripts/Combat/Projectile.cs (hunk 1 above)
Comment (sdudasko) [reply Assets/Scripts/Combat/Projectile.cs:24]: Yeah, there I liked curly brackets style more so else if underneath was looking odd so I added the line

**Pull request merged**